core/src/main/scala/chisel3/Data.scala

```diff
@@ -680,7 +680,7 @@ abstract class Data extends HasId with NamedComponent with SourceInfoDoc {
           if (mod.flatMap(Builder.retrieveParent(_, Builder.currentModule.get)) == Builder.currentModule) =>
       case Some(_: UnconstrainedBinding) =>
       case _ =>
-        throwException(s"operand '$this' is not visible from the current module")
+        throwException(s"operand '$this' is not visible from the current module: (topBinding = ${topBindingOpt})")
```

Comment:
```suggestion
        throwException(s"operand '$this' is not visible from the current module")
```

reverting this change for now